### ISSUE TITLE
G rules seems not working with loguru.

### ISSUE THREAD

**Issue by qiuxiafei** [astral-sh/ruff#13673]
It seems `G` rules only works with built-in `logging` module not loguru. Is it by-design or a bug?

Testing script:
```python
import logging
from loguru import logger

a = 100
logger.info(f"Value of a is {a}")
logging.info(f"Value of a is {a}")

logger.warn("This is a warning")
logging.warn("This is a warning")
```
ruff command (executed in empty project directory without config file) : `ruff check --select G ./test.py`.
The output:
```
  |
4 | a = 100
5 | logger.info(f"Value of a is {a}")
6 | logging.info(f"Value of a is {a}")
  |              ^^^^^^^^^^^^^^^^^^^^ G004
7 |
8 | logger.warn("This is a warning")
  |

test.py:9:9: G010 [*] Logging statement uses `warn` instead of `warning`
  |
8 | logger.warn("This is a warning")
9 | logging.warn("This is a warning")
  |         ^^^^ G010
  |
  = help: Convert to `warning`

Found 2 errors.
[*] 1 fixable with the `--fix` option.
```
ruff points out all errors on logging module but none of loguru's.

ruff version: 0.6.9

**Comment by MichaReiser** [comment]
Yes, this seems intentional. 

Ruff matches the upstream [`flake8-logging-format`](https://pypi.org/project/flake8-logging-format/) linter and its documentation states that it is specific to Python's `logging` module

> Python [logging](https://docs.python.org/3/library/logging.html#logging.Logger.debug) supports a special extra keyword for passing a dictionary of user-defined attributes to include in a logging event.

Our current focus is on linting vanilla Python code, and we only make an exception for very popular third-party libraries that are considered community standards.
